script/uf-unix-format.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+find ../src/ -iname "*.h" -o -iname "*.cpp" -print0 | xargs -0 clang-format --verbose -i --style=file
```

Comment:
So `.clang-format` is in the repository's root directory.

How does clang-format know where to find it?
Does it look for `.clang-format` files all the way to the root directory?
since from where we execute the script it should be located at `../.clang-format`

Reply:
https://stackoverflow.com/questions/46373858/how-do-i-specify-a-clang-format-file